numpy/core/tests/test_records.py

```diff
@@ -341,7 +341,12 @@ def test_tofile_fromfile(self):
                                          formats='f8,i4,a5',
                                          shape=10,
                                          byteorder='<')
-            assert_array_equal(x, a)
+            # check numeric fields within a reasonable
+            # floating point precision match
+            assert_allclose(x['f0'], a['f0'])
```

Comment:
Maybe just use `assert_array_equal` here. seems to work in the failer I showed.

Reply:
As long as there's a good reason for that with floating point numbers -- we don't want to check within i.e., 1e-7 precision, we want an "exact" match?

Reply:
Yes. Serializing to and from a file should yield exact results. We are dumping the bits right? At least I use tofile that way.